src/components/DepositWidget/index.tsx

```diff
@@ -60,9 +68,28 @@ const LinkWrapper = styled.div`
   }
 `
 
+const txOptionalParams: TxOptionalParams = {
+  onSentTransaction: (receipt: Receipt): void => {
+    const notification = <TxNotification txHash={receipt.transactionHash} />
+    if (notification) {
+      toast.info(notification)
```

Comment:
move `const notification` here, no need it create for `else {`

Reply:
The whole _if/else_ check doesn't do anything, for that matter

Reply:
`notification` is always truthy here

Reply:
fixed